src/Routes/Devices/DeviceTable.js

```diff
@@ -178,16 +177,7 @@ const createRows = (
         {
           title: ImageName ? (
             hasLinks ? (
-              <Button
-                variant="link"
-                target-href={pathToImage}
-                onClick={(e) => {
-                  e.preventDefault();
-                  window.location.href = `${pathToImage}`;
-                }}
-              >
-                {ImageName}
-              </Button>
+              <a href={pathToImage}>{ImageName}</a>
```

Comment:
createLink function is not working fro this case  ?

Reply:
Changed to use createLink, this was an remain test before adjust method. Thanks